### DISCUSSION TITLE
Welcome to VulcanUpdate Discussions!

### DISCUSSION THREAD

**Discussion by Luphaestus** [Luphaestus/VulcanUpdate#1] [Announcements]
## 👋 Welcome!
We’re using Discussions as a place specifically for feature requests within our community. We hope that you:

  * Propose new features you’d like to see.
  * Share your ideas and suggestions.
  * Engage with other community members to discuss and refine proposals.
  * Welcome others and remain open-minded as we work together to enhance our offerings 💪.

To get started, comment below with an introduction of yourself and tell us about any feature requests you have in mind.

**Comment by Mesazane** [comment]
Suggestion:
Refreshing shouldn't bounce you back to the TWRP tab
